### DISCUSSION TITLE
Project Manager: Remember project home directory

### DISCUSSION THREAD

**Discussion by caliconiko** [godotengine/godot-proposals#4046] [Editor]
All my godot projects are in one directory, but it isn't in the Documents folder.
So every time I scan a project or make a new project, I have to find my godot projects directory.

I'm thinking it would be nice if you could star directories or just have a list of recent directories.
This is a minor suggestion, but all those times looking for that folder adds up :)

**Comment by havi05** [comment]
The editor has a setting to change this folder.

1. Open a Project.
2. Go to Editorsettings.
3. Click on Filesystem/Directories
4. Set your default project path.

I think it would be useful to add more directories.

![editor_filesystem](https://user-images.githubusercontent.com/71942164/154834446-42a73c05-eda1-4b6e-9403-a1c1b5d96057.png)

**Reply by caliconiko** [reply]
Oh I didn't know you could do that XD. Yeah being able to add multiple directories would be nice

**Reply by theraot** [reply]
I think it would help to have this option available on the Project Manager, so you don't have to open a project to begin with. It would also help people discover it.

Edit: https://github.com/godotengine/godot-proposals/issues/1602

**Reply by Mickeon** [reply]
I believe those options are better off moved to the Project Manager section of the Editor settings and that section accessible from the Project Manager itself.

**Reply by caliconiko** [reply]
> I think it would help to have this option available on the Project Manager, so you don't have to open a project to begin with. It would also help people discover it.

took the words right out of my mouth

**Comment by Calinou** [comment]
There's already a proposal tracking this: https://github.com/godotengine/godot-proposals/issues/1312

See also https://github.com/godotengine/godot-proposals/issues/3560 which is tangentialy related.
